Guess game/Program.cs

```diff
@@ -1,89 +1,126 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Data;
-using System.Linq;
-using System.Runtime.InteropServices;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Guess_game
 {
     class Program
     {
         private static Dictionary<string, string> Riddles = new Dictionary<string, string>
         {
-            {"zalia zole bet ne zole su uodega bet ne pele?" , "agurkas"},
-            { "be ranku be koju duris atidaro?", "vejas"},
-            { "I pakalne letai i kalna greitai?", "snarglys"}
+            {"Žalia žole, bet ne žole su uodega, bet ne pelė?" , "agurkas"},
+            { "Be rankų, be kojų duris atidaro?", "vejas"},
+            { "Į pakalnę lėtai, į kalną greitai?", "snarglys"}
         };
 
+        private static int RiddlesPass = Riddles.Count;
+        private static int GuessLimit = 3;
+        private static int MaxRiddlePoints = 3;
```

Comment:
If you created dedicated variable for max riddle points it should bee used in while loop where you couting actual points, because now chaging `MaxRiddlePoints`to 10 will not make application give 10 points if you will guess riddle from first time (because while loop uses only `GuessLimit` when giving points).  
Additionally having 2 variables `GuessLimit` & `MaxRiddlePoints` is not very convinient, because  if i do `GuessLimit = 4` & `MaxRiddlePoints = 3` then if i guess riddle from 4th time i won't get any points because first 3 guesses lowered my points to 0 already (and if i guess it from 4th time my progress will be equal to -1 and i will loose 1 point out of total instead of getting it).  
I would suggest instead `MaxRiddlePoints` have variable `SingleGuessPoints = 1` in this logic it means that each guess will have 1 point and in case we have guessLimit =3 it means we can have maximum number of 3 points per riddle. And now if i change code to this: `GuessLimit = 3` & `SingleGuessPoints = 2` it means that we can have maximum points of 6 per riddle, what do you think ?